incubator/jaeger/values.yaml

```diff
@@ -1,45 +1,51 @@
 # Default values for jaeger.
 # This is a YAML-formatted file.
+# Jaeger values are grouped by component. Cassandra values override subchart values
 
-# Begin: Override values on the Cassandra/ElasticSearch subchart to customize for Jaeger
-# This chart has been based on the Kubernetes integration found in the following repo:
-# https://github.com/jaegertracing/jaeger-kubernetes/blob/master/production/cassandra.yml
-# https://github.com/jaegertracing/jaeger-kubernetes/blob/master/production-elasticsearch/elasticsearch.yml
+provisionDataStore:
+  cassandra: true
+  elasticsearch: false
```

Comment:
Seems like it was mentioned but never changed. I feel that we should have a single `storage.provisionDataStore` instead of 2 booleans

Reply:
The reason why you need both booleans is because you need to be able to represent 4 states.
Provision a Cassandra instance
Provision a ElasticSearch instance
Use an existing Cassandra instance
Use an existing ElasticSearch instance

Reply:
Totally! But that could be done using the following:

a) Provision a Cassandra instance
```
storage: 
  type: cassandra
  provisionDataStore: true
```

b) Provision a ElasticSearch instance
```
storage: 
  type: elasticsearch
  provisionDataStore: true
```

c) Use an existing Cassandra instance
```
storage: 
  type: cassandra
  provisionDataStore: false
  cassandra:
    host: cassandra
    ....
```

Use an existing ElasticSearch instance
```
storage: 
  type: elasticsearch
  provisionDataStore: false
  elasticsearch: 
    host: elastic
    ...
```

Reply:
Agreed. It can be done that way... it just complicates the requirements.yaml to match the condition for the subchart logic.

Reply:
As I said earlier I'd be happy to do it this way, if someone shows me what the expression would be in the `requirements.yaml`.